android/build.gradle

```diff
@@ -8,22 +8,23 @@ buildscript {
             name 'Google'
         }
     }
-
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.0.1'
+        classpath 'com.android.tools.build:gradle:8.2.2'
     }
 }
 
 android {
-    compileSdkVersion 33
-    buildToolsVersion '33.0.0'
+    compileSdkVersion 34
+    buildToolsVersion '34.0.0'
```

Comment:
@Anush-Shand Since we are bumping Android to 34 please cross check which react-native version we are working with, also note it down if anyone asks. Also cross check release notes of react-native version that we are using.

Reply:
@piyush-kukadiya     
"react-native": "^0.49.5"

This is react native version used and I don't think it has been updated for long. I think it works since we use very basic features of React-Native in the lib

Do we wish to update
The latest version is 0.73.5

Reply:
@Anush-Shand We may update to the latest either with this release or the coming release.

Reply:
@shivamsharma2710 Decided to update to the latest along with the task to support new architecture in the future sprint

Reply:
@Anush-Shand Sure.